tests/utils.js

```diff
@@ -977,6 +977,14 @@ module.exports = {
     return results;
   },
 
+  saveDocIfNotExists: async doc => {
+    try {
+      await module.exports.getDoc(doc._id);
+    } catch (_) {
+      await module.exports.saveDoc(doc);
+    }
+  },
```

Comment:
Nit. It's cheaper to try and save the doc omitting the _rev field. If the doc exists, it'll throw a conflict that you would ignore, and you end up excluding the get request.

Reply:
I think in this case it is better to just leave this how it is.  The problem is that `saveDoc` has a bunch of extra `formDocProcessing` logic.  If you call it with a form doc that does not exist, it will not quickly fail, but will eventually timeout after 6000ms with the error `Timed out looking for details in logs.` because `formDocProcessing` is looking for stuff about the doc in the API logs....

We could roll-our-own `saveDoc` for this function, but that seems unnecessary....

Reply:
I had forgotten about adding the form update watcher in saveDoc. Please disregard :D